torchvision/io/video_reader.py

```diff
@@ -21,11 +18,37 @@ def _has_video_opt() -> bool:
         return False
 
 
+try:
```

Comment:
I wonder if this should be when we `set_video_backend`

Reply:
We could; I have no strong preference.

Reply:
I think it makes more sense as this is only relevant if we set backend to pyav